tests/aws/services/stepfunctions/v2/context_object/test_context_object.py

```diff
@@ -0,0 +1,168 @@
+import json
+
+import pytest
+from localstack_snapshot.snapshots.transformer import RegexTransformer
+
+from localstack.testing.pytest import markers
+from localstack.testing.pytest.stepfunctions.utils import (
+    create_and_record_execution,
+)
+from localstack.utils.strings import short_uid
+from tests.aws.services.stepfunctions.templates.context_object.context_object_templates import (
+    ContextObjectTemplates,
+)
+from tests.aws.services.stepfunctions.templates.services.services_templates import (
+    ServicesTemplates as ST,
+)
+
+
+@markers.snapshot.skip_snapshot_verify(
+    paths=[
+        "$..tracingConfiguration",
+        "$..RedriveCount",
+        "$..RedriveStatus",
+        "$..SdkHttpMetadata",
+        "$..SdkResponseMetadata",
+    ]
+)
+class TestSnfBase:
+    @markers.aws.validated
+    @pytest.mark.parametrize("context_object_literal", ["$$", "$$.Execution.Input"])
+    def test_input_path(
```

Comment:
I considered this but decided against it. 
Parametrisation gets tedious when dealing with combinatorial cases (templates and inputs). We recently tried this [`aws.services.stepfunctions.v2.logs.test_logs.TestLogs`](https://github.com/localstack/localstack/blob/cb81582d84ef612a34bb80c768f55062cf77d01a/tests/aws/services/stepfunctions/v2/logs/test_logs.py#L42) and I don't see it provided any significant benefit -- the task of deciphering the fully qualified test name seems quite inconvenient to me. Maintainability too seems tedious now (not only necessarily wrt updating the snapshots, but also maintaining the ids..) For InputPath and OutputPath, the gains from parametrising also seem slim to me. It would reduce some duplication in the test file, but at the cost of readability and ease of development. What do you think?

Reply:
Yeah agree. Personally I prefer standalone functions over parametrization but I was curious if these tests could be made more succint. Agree that it has too much overhead than what it's worth 👍

Reply:
Replication is indeed unfortunate, and I will continue to look forward to a good solution that can work for us in this space!